jdaviz/core/tests/test_tools.py

```diff
@@ -74,6 +74,11 @@ def test_stretch_bounds(imviz_helper):
 
 
 def test_stretch_bounds_and_spline(imviz_helper):
+
+    # As the histogram randomly samples the array, we should make sure the
+    # values used here are reproducible
+    np.random.seed(42)
```

Comment:
Hmm is there no way to localize it more within the histogram algorithm itself?

Maybe something like `po._obj.stretch_histogram.set_seed(...)` or something?

Reply:
I don't think we actually want to do that right? it's more just for testing? In practice it's probably safer to *not* constrain the random numbers?

Reply:
Yes but I can always do that in the relevant glue tests if needed. I think there shouldn't be a seed set at runtime.